src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -240,12 +242,11 @@ namespace HVACVariableRefrigerantFlow {
         int HRMaxTempLimitIndex = 0;                        // Warning message recurring error index
         int CoolingMaxTempLimitIndex = 0;                   // Warning message recurring error index
         int HeatingMaxTempLimitIndex = 0;                   // Warning message recurring error index
-        std::string FuelType;                               // Fuel type
-        DataGlobalConstants::ResourceType FuelTypeNum;      // Fuel type number
-        Real64 SUMultiplier;                                // exponential timer for mode changes
-        Real64 TUCoolingLoad;                               // total TU cooling load for each VRF system
-        Real64 TUHeatingLoad;                               // total TU heating load for each VRF system
-        bool SwitchedMode;                                  // used to derate capacity/power when system changes operating mode
+        DataGlobalConstants::ResourceType FuelTypeNum = DataGlobalConstants::ResourceType::Invalid; // Fuel type number
+        Real64 SUMultiplier;                                                                        // exponential timer for mode changes
```

Comment:
Initialize the rest of these in the declaration also. Yowzers, there's a lot of them. So maybe not.